 <?xml version="1.0"?>
<launch>

  <node pkg="nodelet" type="nodelet" name="pcl_manager"  args="manager" output="screen"/>
 
 
 
 
 <!-- Run CloudToCloud2Converter -->
  <node pkg="nodelet" type="nodelet" name="CloudToCloud2Converter" args="load cob_env_model/CloudToCloud2Converter pcl_manager" output="screen">
  	<remap from="point_cloud1" to="/full_cloud"/>
  	<remap from="point_cloud2" to="/swissranger/pointcloud2_raw"/>
  	
  </node>  
  
  </launch>